### PR TITLE
Search game bug

### PR DESCRIPTION
**Pull request by Alexandru-Bulai**
# PR 14-04-24
## Overview
In this PR we fix the bug when the user searches for a game 
## Key Changes :
 1) index.js :
    - Instead of targeting the elements that has the data-name and changing the display attribute of that,  we target the container itself and that is what is suppose to be hidden is the inputed value doesn't match 

## Before :

before

## After :

after

### CODE REVIEW

File: src/javaScript/indexDom.js [hunk 1]
```diff
@@ -60,24 +60,24 @@ document.addEventListener('DOMContentLoaded', () => {
...
-    games.forEach((game) => {
+    gridContainer.forEach((game) => {
       const name = game.getAttribute('data-name').toLowerCase()
 
-      if (name.includes(input)) {
-        game.style.display = 'block'
-        visibleGamesCount++
-      } else {
-        game.style.display = 'none'
+      if (game.getAttribute('data-name').toLowerCase() === name) {
```
Comment (vladiiancu) [inline src/javaScript/indexDom.js]: Is name always lowercase as well? 
Maybe force it so you don't depend on the user input as much

**Review by vladiiancu — CHANGES_REQUESTED** [review]

**Review by vladiiancu — COMMENTED** [review]
So you lowercase the data attribute. 
And you lowercase the user input. And then you compare the two values. Or use contains sure

**Comment by Alexandru-Bulai** [comment]
> So you lowercase the data attribute. And you lowercase the user input. And then you compare the two values. Or use contains sure

Thats correct, i convert both strings to lowerCase and then i compare them so see if they match.... is it not a good practice? or you just confirming the logic?

**Comment by vladiiancu** [comment]
Yep. Just confirming

**Review by vladiiancu — APPROVED** [review]

**Pull request merged**